packages/client/hmi-client/src/components/widgets/tera-carousel.vue

```diff
@@ -20,29 +20,30 @@
 				<li v-if="isNumeric && itemCount > 5">(+{{ itemCount }})</li>
 			</ul>
 		</nav>
+		<!-- FIXME: If we are going to use this consider making the chevron navigations work (they should positioned properly)
```

Comment:
can we just get rid of all the commented out stuff 😄

Reply:
Nah we may wanna add it back later. I just didn't fix it because I can't find a carousel in the app anymore! We'll worry about adding them back once we need em.